config/index.js

```diff
@@ -845,8 +845,17 @@ conf.set('smtp.verifySecondaryEmailUrl', conf.get('contentServer.url') + '/verif
 conf.set('isProduction', conf.get('env') === 'prod')
 
 //sns endpoint is not to be set in production
-if (conf.has('snsTopicEndpoint') && conf.get('env') !== 'dev' ) {
+if (conf.has('snsTopicEndpoint') && conf.get('env') !== 'dev') {
   throw new Error('snsTopicEndpoint is only allowed in dev env')
 }
 
+if(conf.get('env')  === 'dev'){
+  if (! process.env.AWS_ACCESS_KEY_ID) {
+    process.env.AWS_ACCESS_KEY_ID = 'foo'
```

Comment:
let's rename `foo` to `DEV_KEY_ID` and the `bar` below to `DEV_ACCESS_KEY`. if we ever see this in the process env if will make more sense where these values came from.